test/unit_tests/js/modules/util/add-email-popup-spec.js

```diff
@@ -2,43 +2,53 @@ const BASE_JS_PATH = '../../../../../cfgov/unprocessed/js/modules/util';
 let addEmailPopup;
 
 const HTML_SNIPPET = `
-<div class="o-email-popup o-email-popup__visible" lang="en" data-popup-label="testPopup">
-    <div class="o-email-popup_header">
+<div class="o-email-popup o-email-signup" lang="en" data-popup-label="testPopup">
+    <div class="o-email-popup_header u-clearfix">
         <div class="close">
-            <a>Close</a>
+            <button class="a-btn a-btn__link">Close</button>
         </div>
     </div>
     <div class="o-email-popup_body">
-        <div class="o-email-signup">
-            <h2>Buying a home?</h2>
-            <form class="o-form o-form__email-signup">
-                <p>Sign up for email tips and info to help you through the process.</p>
-                <div class="m-form-field-with-button">
-                    <div class="form-group">
-                        <input id="form_18" type="email" placeholder="Enter your email address" name="email" class="m-form-field-with-button_field a-text-input" required="">
-                    </div>
-                    <p>
-                        <a href="/owning-a-home-privacy-act-statement/" target="_blank" rel="noopener noreferrer">Privacy Act statement</a>
-                        <br>
-                    </p>
-                    <input class="a-btn a-btn__full-on-xs" type="submit" value="Sign up">
-                </div>
-                <div class="form-group">
-                    <input type="hidden" name="code" value="USCFPB_128">
-                </div>
-            </form>
-            <div class="o-email-signup_footer">
-                <div class="m-notification
-                            m-notification__success
-                            ">
-                    <div class="m-notification_content">
-                        <div class="h4 m-notification_message"></div>
-                    </div>
-                </div>
+        <h2>Buying a home?</h2>
+        <p>Sign up for email tips and info to help you through the process.</p>
+        <form class="o-form o-form__email-signup u-clearfix"
+              id="'o-email-signup_47"
+              method="POST"
+              action="#"
+              enctype="application/x-www-form-urlencoded"
+              novalidate>
+              <div class="m-form-field">
+                  <label class="u-visually-hidden" for="email_47">
+                      Email address
+                  </label>
+                  <input class="a-text-input a-text-input__full"
+                         id="email_47"
+                         name="email"
+                         type="email"
+                         placeholder="Enter your email address"
+                         required>
+              </div>
+
+              <div class="m-btn-group">
+                  <input class="a-btn a-btn__full-on-xs" type="submit" value="Sign up">
```

Comment:
We decided we should use `<button>` instead of `type="submit"` didn't we?

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/4838/commits/75324dfce4ced3b5116c004b5922521ac15ddf4e